string host
string password
---
bool status
